lib/services/bootstrap/pluginBootstrap.js

```diff
@@ -23,12 +23,12 @@ const
   SafeBootstrap = require('./safeBootstrap'),
   errorsManager = require('../../config/error-codes/throw');
 
+// @todo rename to PluginIndexBootstrap
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)